op-guide/binary-deployment.md

```diff
@@ -47,175 +185,314 @@ sha256sum -c tidb-latest-linux-amd64-centos6.sha256
+
+```
+mysql -h 192.168.199.113 -P 4000 -u root -D test
+
+```
 
 ## 多节点集群模式部署
-在生产环境中，我们推荐多节点部署 TiDB 集群，首先请参考[部署建议](./recommendation.md)。
+
+在生产环境中，我们推荐多节点部署 TiDB 集群，首先请参考部署建议。
 
 这里我们使用六个节点，部署三个 PD，三个 TiKV，以及一个 TiDB，各个节点以及所运行服务信息如下：
 
-|Name|Host IP|Services|
-|----|-------|--------|
-|node1|192.168.199.113|PD1, TiDB|
-|node2|192.168.199.114|PD2|
-|node3|192.168.199.115|PD3|
-|node4|192.168.199.116|TiKV1|
-|node5|192.168.199.117|TiKV2|
-|node6|192.168.199.118|TiKV3|
-
-请按如下步骤 **依次启动** PD 集群，TiKV 集群以及 TiDB：
-
-1. 在 node1，node2，node3 依次启动 PD
-
-    ```bash
-    ./bin/pd-server --name=pd1 \
-                    --data-dir=pd1 \
-                    --client-urls="http://192.168.199.113:2379" \
-                    --peer-urls="http://192.168.199.113:2380" \
-                    --initial-cluster="pd1=http://192.168.199.113:2380" \
-                    --log-file=pd.log
-
-    ./bin/pd-server --name=pd2 \
-                    --data-dir=pd2 \
-                    --client-urls="http://192.168.199.114:2379" \
-                    --peer-urls="http://192.168.199.114:2380" \
-                    --join="http://192.168.199.113:2379" \
-                    --log-file=pd.log
-
-    ./bin/pd-server --name=pd3 \
-                    --data-dir=pd3 \
-                    --client-urls="http://192.168.199.115:2379" \
-                    --peer-urls="http://192.168.199.115:2380" \
-                    --join="http://192.168.199.113:2379" \
-                    --log-file=pd.log
-    ```
-
-2. 在 node4，node5，node6 启动 TiKV
-
-    ```bash
-    ./bin/tikv-server --pd="192.168.199.113:2379,192.168.199.114:2379,192.168.199.115:2379" \
-                      --addr="192.168.199.116:20160" \
-                      --data-dir=tikv1 \
-                      --log-file=tikv.log
-
-    ./bin/tikv-server --pd="192.168.199.113:2379,192.168.199.114:2379,192.168.199.115:2379" \
-                      --addr="192.168.199.117:20160" \
-                      --data-dir=tikv2 \
-                      --log-file=tikv.log
-
-    ./bin/tikv-server --pd="192.168.199.113:2379,192.168.199.114:2379,192.168.199.115:2379" \
-                      --addr="192.168.199.118:20160" \
-                      --data-dir=tikv3 \
-                      --log-file=tikv.log
-    ```
-
-3. 在 node1 启动 TiDB
-
-    ```bash
-    ./bin/tidb-server --store=tikv \
-                      --path="192.168.199.113:2379,192.168.199.114:2379,192.168.199.115:2379" \
-                      --log-file=tidb.log
-    ```
-
-4. 使用官方 `mysql` 客户端连接 TiDB
-
-    ```bash
-    mysql -h 192.168.199.113 -P 4000 -u root -D test
-    ```
-
-> 注意：在生产环境中启动 TiKV 时，建议使用 [\-\-config](configuration.md#-c---config) 参数指定配置文件路径，如果不设置这个参数，TiKV 不会读取配置文件。同样，在生产环境中部署 PD 时，也建议使用 [\-\-config](configuration.md#--config) 参数指定配置文件路径。
->
-> 注意：如果使用 `nohup` 在生产环境中启动集群，需要将启动命令放到一个脚本文件里面执行，否则会出现因为 Shell 退出导致 `nohup` 启动的进程也收到异常信号退出的问题，具体参考[进程异常退出](../trouble-shooting.md#tidbtikvpd-进程异常退出)。
+| Name  | Host IP | Services |
+| --- | --- | --- |
```

Comment:
number of hyphens is very meaningful to markdown files.
They indicates approximate width ratio of each columns.
Do to follow the original style.